### ISSUE TITLE
Input containerStyle not working correctly with styled-components

### ISSUE THREAD

**Issue by mauriciord** [react-native-training/react-native-elements#955]
### Description
I need to pass style as object for the prop `containerStyle` of the `Input`, but when i pass style to `containerStyle` with `.attr()` from the [`styled-components`](https://www.styled-components.com/), my `Input` looks horrible comparing to `Input` using object from `Stylesheet` of the `react-native` 

I'm using **react-native-elements@1.0.0-beta2**

```jsx
import React from 'react'
import { View, Text, StyleSheet } from 'react-native'
import styled, { css } from 'styled-components'
import { Input, Button } from 'react-native-elements'
import Colors from '../constants/Colors'

const styles = StyleSheet.create({
  inputContainer: {
    backgroundColor: '#fff',
    borderRadius: 6,
    marginBottom: 10,
    height: 56
  },
  buttonStyle: {
    backgroundColor: '#27AE60',
    borderRadius: 6,
    marginTop: 20
  }
})

const Container = styled.View`
  background-color: ${Colors.tintColor};
  flex: 1;
  justify-content: center;
  align-items: center;
`

const FormInput = styled(Input).attrs({
  containerStyle: styles.inputContainer
})``

const LoginScreen = () => (
  <Container>
    <Input 
      placeholder='E-mail'
      containerStyle={styles.inputContainer}
    />
    <FormInput
      placeholder='Senha'
      secureTextEntry
      containerStyle={styles.inputContainer}
    />
    <Button 
      buttonStyle={styles.buttonStyle}
      text='ENTRAR'
    />
  </Container>
)

export default LoginScreen
```

Screen:
![screen shot 2018-02-28 at 17 50 09](https://user-images.githubusercontent.com/1128701/36812419-ddc349e6-1caf-11e8-8b35-d74c21d657c5.png)

Look the second Input, the font size and another styles has changed.

**Comment by xavier-villelegier** [comment]
Hey @mauriciord @rodrigooler, this has been fixed in #956 and will be released in the `v1.0.0-beta3` very soon 🎉

**Comment by iRoachie** [comment]
Live in [1.0.0-beta3](https://github.com/react-native-training/react-native-elements/releases/tag/v1.0.0-beta3).
